kentikapi/credentials.go

```diff
@@ -0,0 +1,78 @@
+package kentikapi
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+)
+
+// ReadCredentialsFromEnv reads and returns (email, token) pair from environment variables.
+func ReadCredentialsFromEnv() (authEmail, authToken string) {
+	authEmail, _ = os.LookupEnv("KTAPI_AUTH_EMAIL")
+	authToken, _ = os.LookupEnv("KTAPI_AUTH_TOKEN")
+	return authEmail, authToken
+}
+
+// ReadCredentialsFromProfile reads and returns (email, token) pair from profile config file.
+func ReadCredentialsFromProfile() (authEmail, authToken string, err error) {
+	var result map[string]string
+	profile := "default"
```

Comment:
I think that user should have ability to override it with KTAPI_PROFILE env variable